### DISCUSSION TITLE
Mermaid diagrams don't render properly in chromium-based browsers

### DISCUSSION THREAD

**Discussion by joeldeteves** [requarks/wiki#3284] [Error / Bug Report] [answered]
### Version:
**Wiki.js 2.5.170 (latest)**

### Actual behavior
Chromium-based browsers (Chrome, Microsoft Edge, etc) do not render Mermaid diagrams properly. The boxes do not expand to surround the text

![image](https://user-images.githubusercontent.com/33190570/102641089-5833bd00-4110-11eb-8e4b-6a5634fcfb25.png)

### Expected behavior
The diagrams should render properly as they do in Firefox

![image](https://user-images.githubusercontent.com/33190570/102641139-726d9b00-4110-11eb-94e8-817f809972a1.png)

### Steps to reproduce the behavior
1. Create a mermaid diagram
2. Try viewing the diagram in a Chromium based browser e.g. Google Chrome, Microsoft Edge -> doesn't render properly
3. Try viewing the diagram in Firefox -> works correctly

**Comment by ghost** [comment]
You haven't provided the required info about your host! (OS, Wiki.js version, Database engine)

**Comment by joeldeteves** [comment, marked as answer]
Hello, please see https://github.com/mermaid-js/mermaid/issues/1834#issuecomment-804796350

It turned out to be a bug in Chromium browser.

This issue appears to be resolved as of the latest versions of Edge / Chromium.

Thank you,
